packages/x-date-pickers/src/DigitalClock/DigitalClock.tsx

```diff
@@ -171,18 +172,19 @@ export const DigitalClock = React.forwardRef(function DigitalClock(
   const translations = usePickerTranslations();
   const now = useNow(timezone);
 
-  const ownerState = React.useMemo(
-    () => ({ ...props, alreadyRendered: !!containerRef.current }),
-    [props],
-  );
+  const { ownerState: pickerOwnerState } = usePickerPrivateContext();
+  const ownerState: DigitalClockOwnerState = {
+    ...pickerOwnerState,
+    hasDigitalClockAlreadyBeenRendered: !!containerRef.current,
```

Comment:
I'm cleaning the name of this property to keep everything clearly scoped (like `isPickerDisabled` vs `disabled`).